### PR TITLE
refactor editorAccessibility reducers and actions using redux toolkit

### PR DESCRIPTION
**Pull request by PiyushChandra17**
Associated issue: #2042 

Changes:
+ First things first removed all the associated `constants`
+ Used `createSlice` to rewrite the editorAccessibility reducer
+ Exported the automatically generated action creator functions from `actions/editorAccessibility.js`
+ Deleted the existing action creator function 

I have verified that this pull request:

* [x] has no linting errors (`npm run lint`)
* [x] has no test errors (`npm run test`)
* [x] is from a uniquely-named feature branch and is up to date with the  `develop` branch.
* [x] is descriptively named and links to an issue number, i.e. `Fixes #123`

### CODE REVIEW

**Comment by raclim** [comment]
Thanks for your work on refactoring the reducers and actions to use Redux Toolkit! 

I think a few of these PRs aren't passing the tests—would you be able to update those PRs to ensure that they pass them? Thanks!

**Comment by PiyushChandra17** [comment]
> Thanks for your work on refactoring the reducers and actions to use Redux Toolkit!
> 
> I think a few of these PRs aren't passing the tests—would you be able to update those PRs to ensure that they pass them? Thanks!

@raclim I'm quite aware that these few PR's aren't passing the tests as it should be, i'll be soon working on it and make sure that they pass the test going forward. Probably I'll update you soon!

File: client/modules/IDE/reducers/editorAccessibility.js [hunk 1]
```diff
@@ -1,30 +1,38 @@
...
+    updateLintMessage: (state, action) => {
       messageId += 1;
-      return Object.assign({}, state, {
-        lintMessages: state.lintMessages.concat({
-          severity: action.severity,
-          line: action.line,
-          message: action.message,
-          id: messageId
-        })
+      state.lintMessages.push({
```
Comment (lindapaiste) [inline client/modules/IDE/reducers/editorAccessibility.js:16]: You got the mutations right here 👍

Comment (PiyushChandra17) [reply client/modules/IDE/reducers/editorAccessibility.js:16]: @lindapaiste Thanks for the feedback, these are really helpful :)

**Review by raclim — APPROVED** [review]

**Pull request merged**